src/openzaak/components/besluiten/admin.py

```diff
@@ -11,10 +12,27 @@
 from .models import Besluit, BesluitInformatieObject
 
 
+class BesluitInformatieObjectForm(forms.ModelForm):
+
+    class Meta:
+        model = BesluitInformatieObject
+        fields = '__all__'
+
+    def clean(self):
+        cleaned_data = super().clean()
+
+        if not cleaned_data['_informatieobject'] and not cleaned_data['_informatieobject_url']:
```

Comment:
if there would be validation errors (for example because `_informatieobject_url` is not a URL, the key will not be present in `cleaned_data`. Doing a `cleaned_data.get(...)` is more correct. This goes for the other checks as well :)

Reply:
Good point!